bundles/AdminBundle/Resources/public/js/pimcore/document/document.js

```diff
@@ -89,6 +89,10 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
         var saveData = this.getSaveData(only);
 
         if (saveData) {
+            if(this.data.requireEditableValues !== null) {
+                saveData.requireEditableValues = intval(this.data.requireEditableValues);
```

Comment:
I think we should work with `bool` here and let the DB adapter/setter do the required casing.